drivers/sensor/bosch/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ add_subdirectory_ifdef(CONFIG_BMI160 bmi160)
 add_subdirectory_ifdef(CONFIG_BMI270 bmi270)
 add_subdirectory_ifdef(CONFIG_BMI323 bmi323)
 add_subdirectory_ifdef(CONFIG_BMM150 bmm150)
+add_subdirectory_ifdef(CONFIG_BMP388 bmp180)
```

Comment:
Wrong Config

Reply:
I was using the github webinterface before, and change to ssh/local clone to do the work .. to ensure not override the existing files, I edit it.. and introduce the copy/paste mistake. Thanks for catch it.
I fixed it already - part of a interactive rebase coming up.